apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/DataCompactionSparkApp.java

```diff
@@ -20,7 +20,7 @@
  * <p>Example of invocation: com.linkedin.openhouse.jobs.spark.DataCompactionSparkApp --tableName
  * db.testTable --targetByteSize 1048576 --maxByteSizeRatio 0.75 --maxByteSizeRatio 1.8
  * --minInputFiles 5 --maxConcurrentFileGroupRewrites 2 --partialProgressEnabled
- * --partialProgressMaxCommits 10
+ * --partialProgressMaxCommits 10 --deleteFileThreshold 0
```

Comment:
hopefully 0 as default is safe?

Reply:
default is MAX_INT,

we will pass 0 in as an override in short term, via kube cli parameters. it is safe for 12 hour cadence and for minimal tables.

longer term will require a more pragmatic number based on usage. or else, a number derived from DLO strategy